js/snippetPreview.js

```diff
@@ -410,6 +419,10 @@ SnippetPreview.prototype.renderTemplate = function() {
 	} );
 
 	this.element = {
+		measure: {
```

Comment:
Seeing as this is the result of a measurement, naming this `measured` would be far more suitable.